validator-api/src/cache/routes.rs

```diff
@@ -25,3 +26,11 @@ pub(crate) async fn get_rewarded_mixnodes(cache: &State<ValidatorCache>) -> Json
 pub(crate) async fn get_active_mixnodes(cache: &State<ValidatorCache>) -> Json<Vec<MixNodeBond>> {
     Json(cache.active_mixnodes().await.value)
 }
+
+#[get("/mixnodes/rewarded/inclusion-prob/<mixnode_id>")]
+pub(crate) async fn get_probs_mixnode_rewarded(
+    cache: &State<ValidatorCache>,
+    mixnode_id: String,
+) -> Json<HashMap<String, f32>> {
```

Comment:
I think that instead it should be returning a concrete type with explicit fields. Something like
```rust
pub struct InclusionProbabilityResponse {
  rewarded_set: f32,
  active_set: f32,
}
```